code/style.css

```diff
@@ -1,7 +1,425 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+@import url('https://fonts.googleapis.com/css2?family=Alegreya:ital,wght@0,700;1,700;1,900&family=Roboto:wght@300;400;500&display=swap');
+* {
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  margin: 0;
+  padding: 0;
+}
+
+body {
```

Comment:
For mobile/tablet you don't have a hero image/video and on desktop your video is a bit too wide 👇 
<img width="968" alt="Skärmavbild 2021-02-09 kl  12 09 35" src="https://user-images.githubusercontent.com/51529201/107355396-aee5c380-6acf-11eb-9e10-3ba6d215a89f.png">